### DISCUSSION TITLE
Obsidian.md images not importing, user error

### DISCUSSION THREAD

**Discussion by threatseeker** [noraj/OSCP-Exam-Report-Template-Markdown#42] [General]
Hi, noob question. I use Obsidian and would like to use one of the templates to generate my report.  For some reason I cannot get the images to import into pfd when I convert that using pandoc manual script provided here.  I use copy and paste to import my images into my Obsidian notes and place them all in one "img" root folder of the vault.  I'm not sure why the options --resource-path=.:img does not work and I only get the text name of the image in the pdf.  I am sure that is a noob user error.  Any tip would be greatly appreciated.

**Comment by noraj** [comment]
Did you check the FAQ?
https://github.com/noraj/OSCP-Exam-Report-Template-Markdown/blob/master/FAQ.md#how-to-put-images-in-my-report

So both `img/` and `src/img/` should work you can also specify any path by modifying `--resource-path` cf. https://pandoc.org/MANUAL.html#option--resource-path

**Reply by threatseeker** [reply]
Thank you for the tip.  I reviewed the FAQ a few times carefully.  I know what the issue is now.  It is Obsidian image paste option that masks the path to .:img.  When I use the template image place holder path myimage](img/placeholder-image-300x225.png) all works just fine.  I need to figure out how to make Obsidian to do that via paste option and not just put ![[Pasted image 20230104171417.png]].  That would be an amazing time saver creating and generating notes and reports.  Thank you again for your super quick respond, amazing work.  Do you know editor that would do that by default?

**Comment by ismaildawoodjee** [comment]
@j4r0-code  In your Obsidian options in `Files & Links`, the option `Use [[Wikilinks]]` should be **disabled**:

![image](https://github.com/noraj/OSCP-Exam-Report-Template-Markdown/assets/58870992/b82dc759-042d-4d05-8fd9-7b13c3ed1a35)
